app/src/main/java/org/mozilla/reference/browser/browser/BrowserFragment.kt

```diff
@@ -63,6 +67,21 @@ class BrowserFragment : Fragment(), BackHandler, UserInteractionHandler {
     }
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        qrFeature.set(
+            feature = QrFeature(
+                requireContext(),
+                fragmentManager = fragmentManager!!,
+                onNeedToRequestPermissions = { permissions ->
+                    requestPermissions(permissions, REQUEST_CODE_CAMERA_PERMISSIONS)
+                },
+                onScanResult = { pairingUrl ->
```

Comment:
Nice! I'd rename this to `scanresult` though (not pairingUrl).

Reply:
Oops. Yeah, that patch was a bit sloppy. :)